manim/mobject/opengl/opengl_mobject.py

```diff
@@ -1968,7 +2055,13 @@ def put_start_and_end_on(self, start, end):
 
     # Color functions
 
-    def set_rgba_array(self, color=None, opacity=None, name="rgbas", recurse=True):
+    def set_rgba_array(
+        self,
+        color: ParsableManimColor | Iterable[ParsableManimColor] | None = None,
+        opacity: float | Iterable[float] | None = None,
+        name: str = "rgbas",
```

Comment:
At some point, it might be nice to have this as an enum or like `NAMES: TypeAlias = Literal["rgbas", "other-attribute"]`. Maybe something for future PRs?

Reply:
Sounds nice to me